blocks/footer/footer.css

```diff
@@ -304,7 +309,7 @@ a.scroll-to-top:focus {
   }
 
   .footer-bar li:nth-last-child(1) {
-    margin: 0;
+    margin: 0 0 4px;
```

Comment:
You just need the margin-bottom, right?

Reply:
If you look the tree of the css... From where it comes, if you just change the margin bottom you are just restoring what "got lost"
If you set other margins you are setting values for margins that were not set on other viewports